NUnit.proj

```diff
@@ -537,10 +537,14 @@
 
     <!-- Copy root txt files to root -->
     <Copy DestinationFolder="$(CurrentImageDir)" SourceFiles="@(RootFiles)" />
+    
+    <!-- Copy the NuGet files to the nuget directory -->
+    <Copy DestinationFiles="@(NuGetFiles->'$(CurrentImageDir)\nuget\%(Filename)%(Extension)')" 
+        SourceFiles="@(NuGetFiles)" />
```

Comment:
Those files get included in the NuGet package which was referencing them from that location and thus failing to build. I could also change the NuGet package to reference the relative path to root.